reactjs-films-homework/src/components/Header/LanguageToggler/__tests__/LanguageToggler.spec.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { render, screen, fireEvent, cleanup } from '@testing-library/react';
+import LanguageToggler from '../LanguageToggler';
+
+describe('LanguageToggler', () => {
+  afterEach(() => {
+    cleanup();
+  });
+  it('LanguageToggler snapshot', () => {
+    const { asFragment } = render(<LanguageToggler />);
+    expect(asFragment).toMatchSnapshot();
+  });
+
+  it('LanguageToggler Must include text RU or EN', () => {
+    render(<LanguageToggler />);
+    expect(screen.getByText(/en|ru/i)).not.toBeNull();
+  });
+  it('LanguageToggler Must be clickable', () => {
+    render(<LanguageToggler />);
+    fireEvent.click(screen.getByText(/en|ru/i));
+    fireEvent.click(screen.getByText(/ru/i), {
+      target: <div>RU</div>,
+    });
```

Comment:
where the 'expect' statement in this test?)

Reply:
I added the expected statement.
expect(screen.getByText(/ru/i)).toBeInTheDocument();